ceph_deploy/rgw.py

```diff
@@ -404,6 +405,7 @@ def colon_separated(s):
     name = 'rgw.' + s
     if s.count(':') == 1:
         (host, name) = s.split(':')
+    name = validate.alphanumeric(name)
     return (host, name)
 
 
```

Comment:
OK now we need to make a new validate command that supports "." in the name.

This is because rgw names have to start with "rgw." or with "rados." depending on version of ceph.